### DISCUSSION TITLE
Game Backup Monitor v1.3.3

### DISCUSSION THREAD

**Discussion by MikeMaximus** [MikeMaximus/gbm#289] [General]
***Important Notice***

Game Backup Monitor will no longer be receiving any new features or major changes.  However I will continue maintain it and apply fixes as required.

I'd like to thank everyone that made contributions to this project over the years!

***New in 1.3.3***

All Platforms:

- Added missing Simplified Chinese strings. (Thanks @Juijote)
- Fixed an issue with the "Settings" window not closing properly in some cases.
- Fixed an issue with the "Add Game Wizard" window not closing properly after adding a new game.
			
The entire version history of GBM releases is available at http://mikemaximus.github.io/gbm-web/versionhistory.html

<hr /><em>This discussion was created from the release <a href='https://github.com/MikeMaximus/gbm/releases/tag/v1.3.3'>Game Backup Monitor v1.3.3</a>.</em>

**Comment by Darthagnon** [comment]
Wondering, if this is the last feature release, would you like me to do a final update of the [Offline Manual](https://github.com/Darthagnon/gbm-manual) for inclusion in `Game Backup Monitor` >> `Help` >> `Offline Manual` or `Manual`?

**Reply by Darthagnon** [reply]
I've been meaning to update it for ... er... years now, but life's been getting in the way.

**Reply by MikeMaximus** [reply]
Sure!  I'd include it in a maintenance release once you've had some time to update it.
